app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -231,16 +232,19 @@ def unconfirmed_phone?
   end
 
   def phone_view_data
-    {
-      confirmation_for_phone_change: confirmation_for_phone_change?,
+    { confirmation_for_phone_change: confirmation_for_phone_change?,
       phone_number: display_phone_to_deliver_to,
       code_value: direct_otp_code,
       otp_delivery_preference: two_factor_authentication_method,
+      otp_make_default_number: selected_otp_make_default_number,
       voice_otp_delivery_unsupported: voice_otp_delivery_unsupported?,
       reenter_phone_number_path: reenter_phone_number_path,
       unconfirmed_phone: unconfirmed_phone?,
-      account_reset_token: account_reset_token,
-    }.merge(generic_data)
+      account_reset_token: account_reset_token }.merge(generic_data)
+  end
+
+  def selected_otp_make_default_number
+    params&.dig(:otp_make_default_number)
```

Comment:
Isn't passing an argument to `#dig` the same as using `#[]`?

Reply:
Yes it is the same, but dig allows the use of safe navigation operator (&). This makes the code more idiomatic I think.

Reply:
:+1: